static/src/js/components/AccessMethod/__tests__/AccessMethod.test.js

```diff
@@ -50,13 +50,14 @@ const setup = (overrideProps) => {
     ...overrideProps
   }
 
-  render(<AccessMethod {...props} />)
+  const component = render(<AccessMethod {...props} />)
```

Comment:
We should  just be able to use `screen` API to get the elements and not need this wrapper

Reply:
added

Reply:
If you can make it work, you can go ahead and update this.